README.md

```diff
@@ -173,6 +173,11 @@ A curated list of all things anime - streaming/downloading/reading/tracking/foru
 * [trackma](https://github.com/z411/trackma)
 
 #### Mobile
+* [Saikou](https://github.com/saikou-app/saikou)
+* [Cloudstream](https://github.com/recloudstream/cloudstream)
+* [Aniyomi](https://github.com/jmir1/aniyomi)
+* [Animiru](https://github.com/Quickdesh/Animiru)
+* [Kuro](https://github.com/deceptions/no)
```

Comment:
seems the project name is 'no'?

Reply:
when installed the app name is kuro

Reply:
Animiru has better video player and simkl tracking.
Aniyomi's video player is very basic, lacks the option to change subtitle etc.